Models/global_config.hpp

```diff
@@ -0,0 +1,128 @@
+#ifndef ZPSW3_GLOBAL_CONFIG_HPP
+#define ZPSW3_GLOBAL_CONFIG_HPP
+
+#include <cstdint>
+
+#include "AM_ControlAlgorithm.hpp"
+#include "ZP_D_PWMChannel.hpp"
+
+namespace config
+{
+    using percentage_t = float;
+
+        class TempDSHOTDriver : public MotorChannel
+        {
+            public:
+                TempDSHOTDriver() {}
+                void set(uint8_t percent){}
+        };
+    class RCInputDriver{};
```

Comment:
Is this necessary? Seems like a large chunk of this code is examples that should be either documented better with explanations or part of a confluence doc or something.

Reply:
I reorganized it so the example code is isolated and can be removed when an actual model is set up. Keeping an example configuration so there is something to compile for now.